README.md

```diff
@@ -136,40 +136,64 @@ If you'd like to contribute code, get started by cloning the repo and running `n
 
 ```sh
 # Lint your contribution
-grunt lint
+npm run lint
 ```
 
 ```sh
 # Test your contribution
-grunt test
+npm test
+```
+
+### Testing the GitHub Actions workflows
+
+This project's GitHub Actions workflows can be tested locally using [nektos/act](https://github.com/nektos/act), which can be installed with Homebrew:
+
+```sh
+brew install act
+```
+
+To validate the syntax of a workflow:
+
+```sh
+# Validate the publishing workflow, by triggering a 'release' event
+act --dryrun release
 ```
 
 ```sh
-# Lint and test with a single command
-grunt
+# Validate the testing workflow
+act --dryrun push
 ```
 
+To run the testing workflow locally:
+
+```sh
+# Run the testing workflow, with Node.js 18 only
+act push --matrix node-version:18
+```
+
+Add `--verbose` for more output.
+
 ## Support and Migration
 
-Major versions are normally supported for 6 months after their last minor release. This means that patch-level changes will be added and bugs will be fixed. The table below outlines the end-of-support dates for major versions, and the last minor release for that version.
+> [!NOTE]
+> We maintain a [migration guide](MIGRATION.md) to help you migrate between major versions.
 
-We also maintain a [migration guide](MIGRATION.md) to help you migrate.
+When we release a new major version we will continue to support the previous major version for 6 months. This support will be limited to fixes for critical bugs and security issues. If you're opening an issue related to this project, please mention the specific version that the issue affects.
 
-| :grey_question: | Major Version | Last Release | Node.js Versions | Support End Date |
-| :-------------- | :------------ | :----------- | :--------------- | :--------------- |
-| :heart:         | 3             | N/A          | 12+              | N/A              |
-| :hourglass:     | 2             | 2.0.0        | 8+               | 2022-05-26       |
-| :skull:         | 1             | 1.2.1        | 0.10+            | 2020-01-05       |
+The following table lists the major versions available and, for each previous major version, its end-of-support date, and its final minor version released.
 
-If you're opening issues related to these, please mention the version that the issue relates to.
+| Major version   | Final minor version | Node.js support          | Support end date |
+| :-------------- | :------------------ | :----------------------- | :--------------- |
+| `3`             |                     | `>= 12`                  | ✅ Current major version |
+| `2`             | `2.0`               | `8`, `10`                | 2022-05-26       |
+| `1`             | `1.2`               | `0.10`, `0.12`, `4`, `6` | 2020-01-05       |
 
 ## License
 
-Licensed under the [GNU General Public License 3.0](LICENSE.txt).<br/>
-Copyright &copy; 2013–2023, Team Pa11y
+Licensed under the [GNU General Public License 3.0][info-license].  
```

Comment:
Updated to use the link definition (which was pointing to the same file)... but I noticed that:

- the badge and its text `say LGPL` (lines 8 and 204 of this file)
- this line, `LICENSE`, and all the code file headers say `GPL`

Which should it be?

Reply:
mm yes, confusing. I've searched the Slack archives and the Slack archives tell me that LGPL is correct, even though it's the least-used in the codebase 😅

Reply:
Ok thanks @hollsk, I'll make a little PR after this one to update it.